elements/pfe-sass/maps/_typography.scss

```diff
@@ -40,45 +48,90 @@ $pf-type-sizing: (
+/// font-size: var(--pf-c-content--h6--FontSize);
+/// font-size: var(--pf-c-content--FontSize);
+/// font-size: var(--pf-c-content--small--FontSize);
 $pf-type-sizing--content: (
-  content--h1--FontSize:    #{pfe-var(FontSize--3xl )}, // 28px
-  content--h2--FontSize:    #{pfe-var(FontSize--2xl )}, // 26px
-  content--h3--FontSize:    #{pfe-var(FontSize--xl )},  // 20px 
-  content--h4--FontSize:    #{pfe-var(FontSize--lg )},  // 18px
-  content--h5--FontSize:    #{pfe-var(FontSize--md )},  // 16px
-  content--h6--FontSize:    #{pfe-var(FontSize--sm )},  // 14px
+  content--h1--FontSize:    #{pfe-var(FontSize--3xl)}, // 28px
+  content--h2--FontSize:    #{pfe-var(FontSize--2xl)}, // 26px
+  content--h3--FontSize:    #{pfe-var(FontSize--xl)},  // 20px 
+  content--h4--FontSize:    #{pfe-var(FontSize--lg)},  // 18px
+  content--h5--FontSize:    #{pfe-var(FontSize--md)},  // 16px
+  content--h6--FontSize:    #{pfe-var(FontSize--sm)},  // 14px
+  content--FontSize:        #{pfe-var(FontSize--md)},  // 16px  this is for standard paragraph tags
+  content--small--FontSize: #{pfe-var(FontSize--sm)},  // 14px  this is for the small tag
+);
+
+/// Sets of Titles: Mega
+/// @see {mixin} pfe-typography 
+$pf-title-set--mega: ("6xl", "5xl");
 
-  content--xl--FontSize:    #{pfe-var(FontSize--xl )},  // 20px
-  content--lg--FontSize:    #{pfe-var(FontSize--lg )},  // 18px
-  content--FontSize:        #{pfe-var(FontSize--md )},  // 16px
-  content--small--FontSize: #{pfe-var(FontSize--sm )},  // 14px
-  content--xs--FontSize:    #{pfe-var(FontSize--xs )},  // 14px
+/// Sets of Titles: Regular
+/// @see {mixin} pfe-typography 
+$pf-title-set--regular: ("4xl", "3xl", "2xl");
+
+/// Sets of Titles: Small
+/// @see {mixin} pfe-typography 
+$pf-title-set--tiny: (xl, lg, md, sm, xs);
+
+
+/// Title & Text modifiers classes  
+/// @example - css 
+///  font-size: var(--pf-c-title--m-6xl--FontSize);
+///  font-size: var(--pf-c-title--m-5xl--FontSize);
+///  font-size: var(--pf-c-title--m-4xl--FontSize);
+///  font-size: var(--pf-c-title--m-3xl--FontSize);
+///  font-size: var(--pf-c-title--m-2xl--FontSize);
+///  font-size: var(--pf-c-title--m-xl--FontSize);
+///  font-size: var(--pf-c-title--m-lg--FontSize);
+///  font-size: var(--pf-c-title--m-md--FontSize);
+///  font-size: var(--pf-c-title--m-sm--FontSize);
+///  font-size: var(--pf-c-title--m-xs--FontSize);
+///  font-size: var(--pf-c-text--m-xl--FontSize);
+///  font-size: var(--pf-c-text--m-lg--FontSize);
+///  font-size: var(--pf-c-text--m-md--FontSize);
+///  font-size: var(--pf-c-text--m-sm--FontSize);
+///  font-size: var(--pf-c-text--m-xs--FontSize);
+///  font-size: var(--pf-c-text--m-xl--FontSize);
+///  font-size: var(--pf-c-text--m-lg--FontSize);
+///  font-size: var(--pf-c-text--m-md--FontSize);
+///  font-size: var(--pf-c-text--m-sm--FontSize);
+///  font-size: var(--pf-c-text--m-xs--FontSize);
+$pf-type-sizing--modifers: (
+  title--m-6xl--FontSize:   #{pfe-var(FontSize--6xl)},  // 48px   
```

Comment:
You said you needed this interpolation for webdrupal right?  Otherwise none of it is necessary.